### ISSUE TITLE
Python evaluator does not marshal exceptions correctly

### ISSUE THREAD

**Issue by senderista** [uwescience/myria#901]
Here is a Python exception thrown from the Myria Python worker:
```
/usr/local/lib/python2.7/dist-packages/requests-2.18.1-py2.7.egg/requests/__init__.py:80: RequestsDependencyWarning: urllib3 (1.7.1) or chardet (2.0.1) doesn't match a supported version!
  RequestsDependencyWarning)
Traceback (most recent call last):
  File "./reef/global/MyriaPythonWorker.py", line 165, in main
    func = pickle_serializer.read_command(in_file)
  File "./reef/global/MyriaPythonWorker.py", line 138, in read_command
    return cls.loads(unenc)
  File "./reef/global/MyriaPythonWorker.py", line 147, in loads
    return cPickle.loads(obj)
  File "build/bdist.linux-x86_64/egg/myria/__init__.py", line 1, in <module>
  File "build/bdist.linux-x86_64/egg/myria/connection.py", line 9, in <module>
  File "/usr/local/lib/python2.7/dist-packages/requests-2.18.1-py2.7.egg/requests/__init__.py", line 90, in <module>
    from urllib3.exceptions import DependencyWarning
ImportError: cannot import name DependencyWarning
```
And here is the marshaled exception text from the Java logs:
```
edu.washington.escience.myria.DbException: \u0000\u0000\u0002��\n\t\t... 16 more\n\tCaused by: edu.washington.escience.myria.DbException: \u0000\u0000\u0002��\n\t\tat edu.washington.escience.myria.expression.evaluate.PythonUDFEvaluator.readFromStream(PythonUDFEvaluator.java:232)
```

Clearly either the Python worker code is not marshaling exceptions correctly, or `PythonUDFEvaluator.readFromStream()` is not reading the Python exception text correctly.

**Comment by parmitam** [comment]
fixed

**Comment by senderista** [comment]
fixed in #902
